Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -9329,6 +9349,28 @@ a_Player:OpenWindow(Window);
 					},
 					Notes = "Sets the pickup's age, in ticks.",
 				},
+				SetCanCombine =
+				{
+					Params =
+					{
+						{
+							Name = "CanCombine",
+							Type = "boolean",
+						},
+					},
+					Notes = "Sets whether this pickup is allowed to combine with other similar pickups.",
+				},
+				SetLifetime =
+				{
+					Params =
+					{
+						{
+							Name = "SetLifetime",
+							Type = "number",
+						},
+					},
+					Notes = "Sets the total lifespan of this pickup before it despawns, in ticks.",
```

Comment:
Okay, math is okay, as long as there is a way. Just add a note to this function's docs with this info.